src/mon/MgrStatMonitor.cc

```diff
@@ -100,6 +100,11 @@ void MgrStatMonitor::create_pending()
 void MgrStatMonitor::encode_pending(MonitorDBStore::TransactionRef t)
 {
   ++version;
+  if (version < mon->pgmon()->get_last_committed()) {
```

Comment:
maybe we can just fix this in `MgrStatMonitor::create_initial()` instead to this check in every `encode_pending()`.